ui/src/components/Publication/Creation/MainText.tsx

```diff
@@ -358,22 +300,18 @@ const MainText: React.FC = (): React.ReactElement | null => {
             </div>
 
             <div>
-                <Components.PublicationCreationStepTitle text="Short description" />
+                <Components.PublicationCreationStepTitle text="Description" />
```

Comment:
this feels unrelated

Reply:
yep it reverts 

https://github.com/JiscSD/octopus/commit/b169b11b12d5b30807d26dad6de09f97710e613d